release-notes/2023-10/How-to-reset-Dremio.md

```diff
@@ -0,0 +1,54 @@
+# Problem statement
```

Comment:
You spend a lot of time on context before getting to the steps. Is all of this necessary for the user to get started? Your title also tells the user what they are doing but not why.

Reply:
@pvallone Deferring this to your input